package.json

```diff
@@ -44,7 +44,8 @@
   "devDependencies": {
     "concurrently": "5.1.0",
     "stylelint": "13.2.1",
+    "stylelint-config-prettier": "^8.0.1",
```

Comment:
Can we pin this dependency? e.g. remove `^`

Reply:
Good point, done.